demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -128,11 +148,13 @@ def to_dict(self) -> Dict[str, Any]:
         Returns:
             Dict[str, Any]: _description_
         """
+        self.__add_lazy_properties()
```

Comment:
same here, please explain shortly why in the docstring. I'd specifically mention that to_dict is used to create the graph nodes, which is when we want to include the lazy properties.